source/core/types/ut_reporter_base.tps

```diff
@@ -5,6 +5,21 @@ create or replace type ut_reporter_base force authid current_user as object
   constructor function ut_reporter_base(self in out nocopy ut_reporter_base, a_output ut_output default ut_output_dbms_output()) return self as result,
 
   member procedure print_text(self in out nocopy ut_reporter_base, a_text varchar2),
+
```

Comment:
OK, they are console specific.
I could create another abstract on top of `ut_reporter` or just move it to `ut_documentation_reporter`.
Thinking about making the API extensible, maybe another abstract would be good.
Maybe a name like `ut_console_reporter` to keep console specific API?